### DISCUSSION TITLE
HOW webcamoid 8.8 edit virtual camera to 60 fps

### DISCUSSION THREAD

**Discussion by cybercuate** [webcamoid/webcamoid#579] [Q&A]
webcamoid 9 just doesn't work on windows 11, all kinds of crashes. 8.8 works but the only option related to creating a virtual camera is the name. I need a virtual 1920x1080@60fps vcam which I can create with version 9 but it crashes when I try to use it and version 8.8 can't detect the cameras created with v9.

As v8.8 works for me on windows 11 how can I change the vcam created to 60 fps?

**Comment by hipersayanX** [comment]
Isn't possible in Webcamoid 8, that feature was introduced with Webcamoid 9. Anyway, report any bug and crash you have in Webcamoid 9 in the [issues page](https://github.com/webcamoid/webcamoid/issues), a new release is close soon.
